Makefile

```diff
@@ -10,9 +10,8 @@ dev-docs:
 	cd docs && npm install
```

Comment:
Should we add `pre-commit install` to the "dev" target in Makefile to install the pre-commit hooks locally? I had to take that step manually when testing this.

Reply:
absolutely! I thought I had

Reply:
Done - LGTM otherwise!